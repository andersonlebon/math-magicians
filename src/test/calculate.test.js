import calculate from '../mathLogic/calculate';

describe('Check Math calculations', () => {
  it('When Enter AC key', () => {
    const obj = { total: null, next: null, operation: null };
    const restult = calculate(obj, 'AC');
    expect(restult).toStrictEqual(obj);
  });
});
