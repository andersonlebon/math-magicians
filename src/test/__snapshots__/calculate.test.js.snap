// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Calculator test testing the AC Button 1`] = `
Object {
  "next": null,
  "operation": null,
  "total": null,
}
`;
